lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -80,20 +75,107 @@ func (s *subscriberdbServicer) ListSubscribers(ctx context.Context, req *lte_pro
 		return nil, err
 	}
 
+	subProtos, nextToken, err := loadSubProtosPage(req.PageSize, req.PageToken, networkID, apnsByName, apnResourcesByAPN)
+	if err != nil {
+		return nil, err
+	}
+
+	digest, noUpdates := s.getDigestAndNoUpdates(req, networkID, apnsByName, apnResourcesByAPN)
+	if noUpdates {
+		subProtos = []*lte_protos.SubscriberData{}
+	}
+
+	listRes := &lte_protos.ListSubscribersResponse{
+		Subscribers:   subProtos,
+		NextPageToken: nextToken,
+		Digest:        digest,
+		NoUpdates:     noUpdates,
+	}
+
+	return listRes, nil
+}
+
+func (s *subscriberdbServicer) getDigestAndNoUpdates(
+	req *lte_protos.ListSubscribersRequest,
+	networkID string,
+	apnsByName map[string]*lte_models.ApnConfiguration,
+	apnResourcesByAPN lte_models.ApnResources,
+) (*lte_protos.SubscribersDigest, bool) {
+
+	digest, noUpdates := req.Digest, false
+	// This functionality is currently placed behind a feature flag.
+	if s.flatDigestEnabled && req.PageToken == "" {
```

Comment:
Flip this if -- instead of nesting the functionality inside the if block, use an early return